hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -201,9 +201,40 @@ def hubble_versions():
     '''
     Report version of all hubble modules as query
     '''
-    versions = {'nova': __version__,
-                'nebula': __version__,
-                'pulsar': __version__,
-                'quasar': __version__}
+    versions = {}
+
+    # Nova
+    if 'hubble.version' in __salt__:
+        versions['nova'] = __salt__['hubble.version']()
+    else:
+        versions['nova'] = None
+
+    # Nebula
+    versions['nebula'] = version()
+
+    # Pulsar
+    if salt.utils.is_windows():
+        try:
+            sys.path.insert(0, os.path.dirname(__salt__['cp.cache_file']('salt://_beacons/win_pulsar.py')))
+            import win_pulsar
+            versions['pulsar'] = win_pulsar.__version__
+        except:
+            versions['pulsar'] = None
+    else:
+        try:
+            sys.path.insert(0, os.path.dirname(__salt__['cp.cache_file']('salt://_beacons/pulsar.py')))
+            import pulsar
+            versions['pulsar'] = pulsar.__version__
+        except:
+            versions['pulsar'] = None
+
+    # Quasar
+    try:
+        sys.path.insert(0, os.path.dirname(__salt__['cp.cache_file']('salt://_returners/splunk_nova_return.py')))
+        import splunk_nova_return
+        versions['quasar'] = splunk_nova_return.__version__
+    except:
+        versions['quasar'] = None
+
```

Comment:
Another section that I think came from hubble-salt. In hubble we just need the 4 lines that were deleted.